pl_bolts/models/rl/double_dqn_model.py

```diff
@@ -34,21 +34,6 @@ class DoubleDQN(DQN):
         trainer = Trainer()
         trainer.fit(model)
 
-    Args:
-        env: gym environment tag
-        gpus: number of gpus being used
-        eps_start: starting value of epsilon for the epsilon-greedy exploration
-        eps_end: final value of epsilon for the epsilon-greedy exploration
-        eps_last_frame: the final frame in for the decrease of epsilon. At this frame espilon = eps_end
-        sync_rate: the number of iterations between syncing up the target network with the train network
-        gamma: discount factor
-        lr: learning rate
-        batch_size: size of minibatch pulled from the DataLoader
-        replay_size: total capacity of the replay buffer
-        warm_start_size: how many random steps through the environment to be carried out at the start of
-            training to fill the buffer with a starting point
-        sample_len: the number of samples to pull from the dataset iterator and feed to the DataLoader
-
```

Comment:
Deleted these lines to fix duplicate `Parameters` sections in docs. Before the change, there were two `Parameters` section in docs from both these lines and from docstring of `DQN` which is the base class for `DoubleDQN`. See https://pytorch-lightning-bolts.readthedocs.io/en/latest/reinforce_learn.html#double-dqn-results.

Reply:
if we remove it, pls add a cross-reference to the source where it is described

Reply:
I am not sure how to add a cross-reference to the source. I would be grateful if you could possibly show me how to do so.

Reply:
let's write something like for the details of the arguments see :ref:~.... (not sure what is the syntax now, but you can find it elsewhere in our docs :])

Reply:
Sure, thank you! Do we still want a cross-reference though because we have the details of the arguments at `DoubleDQN` even after deleting these lines from `DoubleDQN` docstring. `DQN`'s docstring is automatically rendered as `DoubleDQN`'s docs.
Here is the docs rendered after the deletion:
 ![20201023-044044](https://user-images.githubusercontent.com/20610905/96921790-f4af5980-14e9-11eb-84e3-31d44f3df1ff.png)